src/component/form/FormInput.js

```diff
@@ -22,7 +22,7 @@ const validationSchema = Yup.object({
     .max(100000, "Number is less than 100000")
     .required("Required"),
   emailField: Yup.string()
-    .matches(/^\w+@gmail.com$/i, "Email is invalid")
+    .matches(/^\w+([._-]?\w+)*@\w+([._-]?\w+)*(\.\w{2,})$/i, "Email is invalid")
     .required("Required"),
```

Comment:
If I have a task to validate the email like this,I will use this one :
`` /^[a-zA-Z0-9!#$%&''*+/=?^_`{}~@."\-\s]*$/ ``

Reply:
.matches(
    /^[a-zA-Z0-9!#$%&''*+/=?^_`{}~@."\-\s]*$/,
    'Please enter valid email address'
  )